lib/LMSManagers/LMSFileManager.php

```diff
@@ -33,15 +33,15 @@ public function GetFileContainers($type, $id)
             return null;
         }
 
-        $result = $this->db->GetAll('SELECT c.*, u.name AS creatorname
+        $result[$type] = $this->db->GetAll('SELECT c.*, u.name AS creatorname
 			FROM filecontainers c
 			LEFT JOIN vusers u ON u.id = c.creatorid
 			WHERE c.' . $type . ' = ?', array($id));
         if (empty($result)) {
             return null;
         }
 
-        foreach ($result as &$container) {
+        foreach ($result[$type] as &$container) {
```

Comment:
Tutaj ustawianie pod indeksem takim jak typ zasobu nie jest potrzebne - to można zrobić na zewnątrz tylko przy netdevinfo.